### ISSUE TITLE
Init error on django 1.9.7

### ISSUE THREAD

**Issue by jorrete** [safwanrahman/django-webpush#8]
I am having errors to runserver on django 1.9.7:
```
"INSTALLED_APPS." % (module, name)
RuntimeError: Model class django.contrib.contenttypes.models.ContentType doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
Performing system checks...
```
It seems that django version  doesn't play well with import on __init__.py. Once I move the functions to utils.py everything seems fine.

**Comment by safwanrahman** [comment]
Thanks for filling the issue. I have not tested with django 1.9.x yet.
So seems like its a bug, and need to be fixed in order to use with django 1.9.x and onwards.

I will look over into it as soon as I finish implementing the chrome support #4 feature.

If anyone can help on this by sending a pull request, that would be much appreciated. :+1:

**Comment by safwanrahman** [comment]
So I wil start working on it.
sorry for the delay

**Comment by jamaalscarlett** [comment]
should be fixed by https://github.com/safwanrahman/django-webpush/pull/10

**Comment by safwanrahman** [comment]
@jorrete Can you please confirm if your problem is fixed by #10 ? So I can close the issue

**Comment by safwanrahman** [comment]
I have tested that and it seems that #10 fix the problem. so closing in favor of #10
Thanks @jamaalscarlett @jorrete
